module-2/backend/week_four.md

```diff
@@ -10,14 +10,45 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+Cookies are saved information that the website keeps in order to "remember" a User.
+Google definition: "An HTTP cookie (also called web cookie, Internet cookie, browser cookie or simply cookie) is a small piece of data sent from a website and stored on the user's computer by the user's web browser while the user is browsing"
+
 * What’s the difference between a session and a cookie?
+A session is more of a timeframe of dialogue between two devices. 
```

Comment:
a session is hashed while the cookie is usually in plaintext. we use sessions to store the data we want to track because it's sensitive!